apisix/cli/ngx_tpl.lua

```diff
@@ -120,9 +120,12 @@ http {
     # put extra_lua_path in front of the builtin path
     # so user can override the source code
     lua_package_path  "{*extra_lua_path*}$prefix/deps/share/lua/5.1/?.lua;$prefix/deps/share/lua/5.1/?/init.lua;]=]
+                       .. [=[/usr/share/lua/5.1/?.lua;/usr/share/lua/5.1/?/init.lua;]=]
```

Comment:
I think we need to update the `lua_package_path` of `stream`

Reply:
Exactly. I'v done.